<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResolutionDialog</class>
 <widget class="QDialog" name="ResolutionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>261</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Choose resolution</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QRadioButton" name="useCurrent">
     <property name="text">
      <string>Use current resolution (%RES%)</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="useSpecified">
     <property name="text">
      <string>Use resolution specified below:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="formAlignment">
      <set>Qt::AlignHCenter|Qt::AlignTop</set>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="widthLabel">
       <property name="text">
        <string>Width:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="heightLabel">
       <property name="text">
        <string>Height:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="bppLabel">
       <property name="text">
        <string>Color depth:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="refreshLabel">
       <property name="text">
        <string>Refresh rate:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="widthBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="suffix">
        <string> px</string>
       </property>
       <property name="maximum">
        <number>2048</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="heightBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="suffix">
        <string> px</string>
       </property>
       <property name="maximum">
        <number>1152</number>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="bppBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="suffix">
        <string> bpp</string>
       </property>
       <property name="maximum">
        <number>32</number>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="refreshBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="suffix">
        <string> Hz</string>
       </property>
       <property name="maximum">
        <number>256</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>useSpecified</sender>
   <signal>toggled(bool)</signal>
   <receiver>widthBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useSpecified</sender>
   <signal>toggled(bool)</signal>
   <receiver>heightBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useSpecified</sender>
   <signal>toggled(bool)</signal>
   <receiver>bppBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>166</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useSpecified</sender>
   <signal>toggled(bool)</signal>
   <receiver>refreshBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>166</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ResolutionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>202</y>
    </hint>
    <hint type="destinationlabel">
     <x>130</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ResolutionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>202</y>
    </hint>
    <hint type="destinationlabel">
     <x>130</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
